src/plugins/intel_cpu/src/emitters/snippets/aarch64/cpu_generator.cpp

```diff
@@ -179,7 +180,9 @@ CPUTargetMachine::CPUTargetMachine(dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
     jitters[op::v1::Minimum::get_type_info_static()] = CREATE_CPU_EMITTER(jit_minimum_emitter);
     jitters[op::v1::Mod::get_type_info_static()] = CREATE_CPU_EMITTER(jit_mod_emitter);
     jitters[op::v1::Multiply::get_type_info_static()] = CREATE_CPU_EMITTER(jit_multiply_emitter);
+    jitters[op::v1::Power::get_type_info_static()] = CREATE_CPU_EMITTER(jit_power_static_emitter);
```

Comment:
As I wrote in the [comment](https://github.com/openvinotoolkit/openvino/issues/28162#issuecomment-2709686795), we have optimization [pass](https://github.com/a-sidorova/openvino/blob/385cfee24a95027c6b303edad1a30c1f58d49d21/src/common/snippets/src/pass/convert_power_to_powerstatic.cpp#L11) which transform `op::v1::Power` with constant scalar second input to `snippets::op::PowerStatic`. In this case there should be mapping `snippets::op::PowerStatic` on `jit_power_static_emitter`. Please correct it here.

Reply:
Should PowerStatic also be used in `supported_op` lambda? Or should that be `op::v1::Power`?

Reply:
`is_supported_op` is called during the tokenization stage when the model contains only `op::v1::Power` (ops from only  OpenVINO opsets - `ov::op::vX`). After tokenization stage there will be Optimization stage where `op::v1::Power` with constant scalar second input will be converted to `snippets::op::PowerStatic`.

It means that `supported_op` should check only for `op::v1::Power`